homeassistant/components/senseme/light.py

```diff
@@ -51,7 +51,8 @@ def __init__(self, device: SensemeDevice, name: str) -> None:
     def _async_update_attrs(self) -> None:
         """Update attrs from device."""
         self._attr_is_on = self._device.light_on
-        self._attr_brightness = int(min(255, self._device.light_brightness * 16))
+        if self._device.light_brightness is not None:
```

Comment:
If device brightness is None, should `self._attr_brightness` be set to `None` ?

Reply:
AFAICT It's just missing when the device reconnects and it will come in the next update. So we need to ignore None